README.md

```diff
@@ -30,11 +30,28 @@ git clone https://github.cerner.com/terra/terra-ui.com.git
 bundle install && npm install
 ```
 
+<details>
+<summary>If bundle install fails</summary>
+An error occurred while installing pg (0.20.0), and Bundler cannot continue.Make sure that gem install pg -v '0.20.0' succeeds before bundling., then install posgresql and try again.
+
+brew install postgresql (mac)
+
+</details>
+
 ### Run Application
 ```
 npm run start
 ```
 
+<details>
+<summary>If npm run start fails</summary>
+sh: foreman: command not found, then you need install the foreman gem.
+
+gem install foreman
```

Comment:
do you need to install foreman globally or can your package.json just be `bundle exec foreman` since foreman is in the gemfile